_posts/2024-01-25-WASI-Preview2.md

```diff
@@ -0,0 +1,206 @@
+---
+title: WASI Preview 2 Launched
+author: "Dan Gohman"
+github_name: sunfishcode
+---
+> Original post from https://blog.sunfishcode.online/wasi-preview2/
+
+The WASI Subgroup has just voted to launch WASI Preview 2! This blog post
+is a brief look at the present, past, and future of WASI.
+
+## The present
+
+The Subgroup voted to launch Preview 2!
```

Comment:
```suggestion
The Subgroup voted to launch WASI 0.2, also known as WASI Preview 2 or WASIp2!
```

Reply:
If this is the best place to call out that 0.2 was formerly known as Preview 2 then OK, but I proposed doing that in the opening paragraph in my comment above.  Let me know what you think.

Reply:
Sounds good. I've now updated the edits.